02LessonTwo/in-class-project/index.html

```diff
@@ -1,10 +1,26 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>A&R Dog Walking</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-
+  <h1 style="text-align: center;"><u>A&R Dog Walking</u></h1>
+    <img src="https://petride.files.wordpress.com/2011/07/dog_walker.jpg" src="img-align: center;">
+  <h2><u>About Us</u></h2>
+    <p>We're avid dog lovers wanting to walk your dogs!</p>
+   <p>Whether you have longer hours at work, or simply not enough time to manage all the chores, we've got you covered.<br></br>
+   Austin Dog Butler is here to serve you. We give care your dog deserves from the home your dog prefers.</p>
+   <h3><u>Services</u></h3>
+    <ul>
+      <li>Dog walking</li>
+      <li>Dog cleaning</li>
+      <li>Dog sitting</li>
+      <li><a href="http://www.tomlinsons.com/anesthesia-free-teeth-cleaning/"target="_blank">Teeth cleaning</a></li>
```

Comment:
don't forget to space out the end of the attribute **href**="...." (space) **target**="..."